### PR TITLE
[Docs] March 2024 Release Notes

### PR DESCRIPTION
**Pull request by nataliekwong**
# What
Release Notes for March 2024 + some misc updates to platform docs

## Content
- OIDC Support
- MSSQL Certification
- Redshift V2
- Docker & Helm breaking change
- Stream-scoped breaking changes for Cloud
- Various connector improvements

### CODE REVIEW

File: docs/release_notes/march_2024.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# February 2024
```
Comment (natikgadzhi) [inline docs/release_notes/march_2024.md:1]: Should this be March? 

```suggestion
# March 2024
```

File: docs/release_notes/march_2024.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- A major upgrade to our Docker and Helm deployments, which simplifies how external logs are configured. Learn more about the specific changes in our [migration guide](/deploying-airbyte/on-kubernetes-via-helm#migrate-from-old-chart-to-airbyte-v0520-and-latest-chart-version).
+
+
+- Our major version upgrades (Airbyte Cloud only) now only require manual upgrading when you are actively syncing a stream that has changed. Otherwise, syncs will continue as is and the version will be upgraded automatically for you.
+
+## Connector Improvements
+
+In addition to our MS-SQL certification, we also released a few notable Connector improvements:
+
+- We released several connector builder enhancements,including support for raw YAML blocks and the ability to reference custom python components to enable running connectors that use custom components. You can also modify the start date when testing and adjust page/slice/record limits.
```
Comment (natikgadzhi) [inline docs/release_notes/march_2024.md:28]: Hmm, I would love to document those better, but I don't have anything specific in mind to link to.

Nit: ability to reference custom components will only work in development / local Builder. 

Other builder improvements:
- Fixed bugs in page size and interpolation inputs: https://github.com/airbytehq/airbyte-platform-internal/pull/11926
- Switching from YAML to UI is faster now: https://github.com/airbytehq/airbyte-platform-internal/pull/11848
- Fixed some annoying layout Builder bugs: https://github.com/airbytehq/airbyte-platform-internal/pull/11695

Comment (nataliekwong) [reply docs/release_notes/march_2024.md]: Added all three!

**Review by natikgadzhi — APPROVED** [review]
LGTM from my side.

**Comment by techxorcist** [comment]
Good day. The PR mentions breaking changes to Docker but the material referred to in the Release Notes seems limited to Helm chart changes.  Could this [discussion](https://github.com/airbytehq/airbyte/discussions/36984) be related to the breaking change and what is that change in relation to Docker installs? Thank you.

**Comment by nataliekwong** [comment]
Hi @techxorcist , could you try the suggestion here? It looks like it's related to our rootless changes.  https://github.com/airbytehq/airbyte/issues/36783#issuecomment-2037707927

**Comment by techxorcist** [comment]
Thank you @nataliekwong

**Comment by nataliekwong** [comment]
Anything you'd like changed @katmarkham ?

**Pull request merged**